cmd/createredis.go

```diff
@@ -26,19 +26,24 @@ import (
 
 var redisConfName string
 
-func redisParseConfig(tm *tcloudredis.TcloudRedis) error {
+func redisParseConfig() (*tcloudredis.TCloudRedis, error) {
 	// Open our jsonFile
```

Comment:
if you return a tcloudredis.TCloudRedis in this function, can we change `redisParseConfig` name to something like `createRedisInstance`

Reply:
I'll rename MySQL to match too